venus-worker/src/sealing/worker/task/planner/sealer.rs

```diff
@@ -169,7 +169,7 @@ impl<'c, 't> Sealer<'c, 't> {
 
         let sector = match maybe_allocated {
             Some(a) => a,
-            None => return Ok(Event::Retry),
+            None => return Ok(Event::Idle),
```

Comment:
`Event::Idle` should cover  `maybe_allocated` ?

Reply:
resolved.